sdks/workflow-browser-sdk/.eslintrc.cjs

```diff
@@ -8,4 +8,14 @@ module.exports = {
     project: ['./tsconfig.eslint.json'],
   },
   ignorePatterns: ['.eslintrc.cjs'],
+  rules: {
+    '@typescript-eslint/no-unused-vars': [
```

Comment:
Worried this `@typescript-eslint/no-unused-vars` rule will result in more unused code long term. For places which an unused variable makes sense disabling the rule only for that line is something I'd feel more comfortable with.

Reply:
the rule is not disabled, it allows prepending underscore to unused variables;
there are some valid usages of this rule, like dropping something from arrays eg: `const [_drop, _me, useMe] = ...` which looks better than `const [,, useMe]` or skipping some stuff in arguments. 

I'd leave it, it will be visible in in PRs - however i have no strong opinion on it

Reply:
Exactly what I meant, the valid usages are great. Problem is there is no way of limiting the rule to destructuring. I don't mind it in the PR either, some of the instances where we use _ might require revisiting.

Reply:
Maybe a more specific prefix will help

Reply:
"drop_", "unused_"? now when writing down it feel a bit strange - im good with Omri's suggestion

Reply:
i'll drop the shared config as well then